### ISSUE TITLE
Example of setting blog post titles to include date doesn't change to include date

### ISSUE THREAD

**Issue by bdjulian** [deargle-classes/security-analytics-assignments#4]
[Lab -- Github, Markdown, and You](https://classes.daveeargle.com/security-analytics-assignments/labs/lab-github-and-github-pages.html)

Under [Using Jekyll for Blogging](https://classes.daveeargle.com/security-analytics-assignments/labs/lab-github-and-github-pages.html#use-jekyll-for-blogging):

Adding blog post dates to the blog index. The example snippet of how to include the post date is shown:
```
{{ post.title }} | {{ post.date | date_to_string: "ordinal", "US" }}
```
But then the it is not added to blog.md in the next example.

```
---
---

<h1>Blog</h1>

<ul>
 {% for post in site.posts %}
 <li>
   <a href='{{ post.url | relative_url }}'>{{ post.title }}</a>
 </li>
 {% endfor %}
</ul>
```

**Comment by deargle** [comment]
:+1: Want to open a PR that fixes this?

**Comment by bdjulian** [comment]
Will do

**Comment by deargle** [comment]
Closed by #5

**Comment by deargle** [comment]
Thanks! 🎉
